artemis-server/src/main/java/org/apache/activemq/artemis/core/postoffice/impl/PostOfficeImpl.java

```diff
@@ -1582,7 +1582,7 @@ public void run() {
       public void run() {
          for (Queue queue : getLocalQueues()) {
             if (!queue.isInternalQueue() && QueueManagerImpl.isAutoDelete(queue) && QueueManagerImpl.consumerCountCheck(queue) && QueueManagerImpl.delayCheck(queue) && QueueManagerImpl.messageCountCheck(queue)) {
-               QueueManagerImpl.deleteAutoCreatedQueue(server, queue);
+               QueueManagerImpl.deleteAutoDeleteQueue(server, queue);
```

Comment:
I'm confused.. is that a typo on the name?

Reply:
its because the name gives meaning it only affected auto created queue, where thats not true. you want delete on any queue thats auto delete.

Reply:
Do you mind if I rename this to performAutoDeleteQueue?

I can amend it myself if you agree.

Reply:
sure. just wanted better naming that was all. happy with your suggestion.

Reply:
ok, I amended it during merge with a temporary branch (used scripts ./scripts/checkout-branch.sh, amended, and ./scripts/merge-branch.sh)

it will appear on the git log as if you did it. (it's just better than coming back and forth asking you to do it).